src/main/java/core/basesyntax/BallColors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum BallColors {
```

Comment:
enum name should be singular
```suggestion
public enum Color {
```